modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -459,4 +459,98 @@ definition {
 		}
 	}
 
+	@description = "LPS-178783 - Verify that it's possible to send an e-mail notification with Action (Standalone)"
+	@priority = 5
+	test CanSendEmailViaActionStandalone {
+		property custom.properties = "feature.flag.LPS-173894=true";
+		property portal.acceptance = "true";
+		property test.smtp.server.enabled = "true";
```

Comment:
You're gonna need an additional property here to skip portal instances, since we need to change the notification check interval in System Settings. You can't access System Settings in virtual instances.

Reply:
Fixes sent, this is the only one I am confused about (I see `property test.name.skip.portal.instance =` but I am not sure what to set it to)

Reply:
You set it to `TestCaseFileName.TestName`. So in this case, it would be `ObjectNotifications.CanSendEmailViaActionStandalone`

Reply:
I just found out that you don't actually need to configure the interval since by default, it'll send the notification immediately. You don't need this property, @bronsonsoda